lib/routes/account.js

```diff
@@ -498,12 +498,14 @@ module.exports = function (
               // If the account is already verified, they may be e.g.
               // clicking a stale link.  Silently succeed.
               if (account.emailVerified) {
+                log.activityEvent('account.already_verified', account.uid, request)
                 return true
               }
               if (!butil.buffersAreEqual(code, account.emailCode)) {
                 throw error.invalidVerificationCode()
               }
               log.event('verified', { email: account.email, uid: account.uid, locale: account.locale })
+              log.activityEvent('account.verified', account.uid, request)
```

Comment:
we have another .event call above. However Andy added these "activityEvent"s, is it okay to have these together or should be find a better way to organize this?